### ISSUE TITLE
[Accessibility]: Filtered connection count changes from bottom to top when text field has input

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35320]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33514

### Problem description

When the text input field has no content in a Filtered Connection, the count of elements is at the very bottom, which is difficult to navigate to cleanly (why should I have to go to the bottom). It moves to the top once there is text in the text input field. See screenshots

### Expected behavior

Count should be at the top in both instances for ease of access and conveying information to the screen reader.

### Additional details

![image](https://user-images.githubusercontent.com/18282288/167930029-01780739-cddc-4b4a-bc74-b9b797597e32.png)

![image](https://user-images.githubusercontent.com/18282288/167930076-82d4b542-f1d8-41c3-a77d-d28ac220cbf7.png)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by felixfbecker** [comment]
It is a weird inconsistency, but I don't think it's a WCAG violation.
